enum/Analytics.ts

```diff
@@ -52,6 +52,7 @@ export const Events: Record<string, IAnalyticsEvent> = {
 		action: 'successful',
 		properties: {
 			queue_time: '',
+			sf_chat_key: '',
```

Comment:
@csmith14 do you happen to know the status of this on Molly's end?  Does she need special handling to accomodate two properties, and is that ready at this point?  Thanks.

Reply:
Yes, Molly has already merged (not sure if deployed yet but I believe so) changes to accommodate this new event structure in the Ciao analytics parser.